tests/server_test.py

```diff
@@ -15,6 +15,40 @@ def test_elegant_run_file(fc):
     _get_file(fc, _run_elegant(fc), "downloadRunFile")
 
 
+def test_error_logging(fc):
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkeq
+
+    r = fc.sr_post("errorLogging", PKDict(message="error message"))
```

Comment:
temporary is unnecessary

Reply:
1a08a18cb5e9c42888b493a7621bd56f9ac7faea